orc8r/cloud/configs/obsidian.yml

```diff
@@ -9,3 +9,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# When true, poll and merge Swagger specs at runtime
```

Comment:
Prefer Go-style doc comments pretty much everywhere -- i.e., prefix the doc comment with the name of the object it's documenting